dot/network/service.go

```diff
@@ -87,6 +87,13 @@ var (
 		Name:      "outbound_total",
 		Help:      "total number of outbound streams",
 	})
+	processStartTimeGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "substrate", // Note: this is using substrate namespace because that is what zombienet uses
+		//  to confirm nodes have started TODO: consider other ways to handle this.
```

Comment:
Is it worth opening an issue for this todo?

Reply:
Yes, I've created issue #3205 to track this.